csrc/ir/nodes.cpp

```diff
@@ -5848,6 +5919,40 @@ std::vector<PolymorphicValue> GroupedMmaOp::evaluate(
   const auto& mat2 = inputs[1].as<at::Tensor>();
   const auto& offsets = inputs[2].as<at::Tensor>();
 
+  std::optional<at::Tensor> alpha = std::nullopt;
```

Comment:
```suggestion
  at::Tensor alpha;
```
and later `alpha.defined()`